src/elm/Pages/Build/Logs.elm

```diff
@@ -294,6 +333,27 @@ downloadFileName org repo buildNumber resourceType resourceNumber =
     String.join "-" [ org, repo, buildNumber, resourceType, resourceNumber ]
 
 
+{-| logSizeLimit : returns the maximum size for log data, in bytes.
+-}
+logSizeLimit : Int
+logSizeLimit =
+    Util.megabyte * 5
```

Comment:
here we define the actual size limit.

should this be a runtime flag, or should this just be imposed by the code.